sycl/plugins/level_zero/usm_allocator.cpp

```diff
@@ -148,7 +148,7 @@ static class SetLimits {
     // Example of usage:
     // SYCL_PI_LEVEL_ZERO_USM_ALLOCATOR=1;32M;host:1M,4,64K;device:1M,4,64K;shared:0,0,2M
 
-    auto GetValue = [](std::string &Param, size_t Length) {
+    auto GetValue = [](std::string &Param, size_t Length, size_t InVal) {
```

Comment:
Return with reference?
```suggestion
    void GetValue = [](std::string &Param, size_t Length, size_t &Setting) {
```